scss/components/_discovery-panel.scss

```diff
@@ -0,0 +1,123 @@
+.oo-discovery-toaster-container-style {
+  @include absolute-center;
+  position:relative;
+  background-color: transparent;
+  height: auto;
+  max-height: 370px;
+  width: 100%;
+  overflow: hidden;
+  white-space: nowrap;
+  margin: auto;
+}
+@include apply-to-safari-only ('.oo-discovery-toaster-container-style') {
+  min-height: 270px;
+}
+@include apply-to-ie11-only ('.oo-discovery-toaster-container-style') {
+  position: absolute;
```

Comment:
@chibzu there were some issues from PR #480 with discover content being cutoff, I applied the IE11 fix to a IE11 only mixin